ambry-frontend/src/main/java/com.github.ambry.frontend/AmbrySecurityServiceFactory.java

```diff
@@ -31,7 +31,7 @@
   private final UrlSigningService urlSigningService;
 
   public AmbrySecurityServiceFactory(VerifiableProperties verifiableProperties, ClusterMap clusterMap,
-      AccountService accountService, UrlSigningService urlSigningService,
+      AccountService accountService, UrlSigningService urlSigningService, IdSigningService idSigningService,
```

Comment:
is it worth keeping `idSigningService` as a variable inside class for future use?

Reply:
I think I prefer leaving it out, since the variable is easy to add later on and I don't want to add additional unused code.